lib/plugin-manager.coffee

```diff
@@ -135,11 +138,14 @@ class PluginManager
 
   # Working with autocomplete
   registerAutocompleteProviders: ->
-    if atom.packages.isPackageLoaded('autocomplete-plus')
-      atom.packages.activatePackage('autocomplete-plus')
-        .then (pkg) =>
-          @autocompleteModule = pkg.mainModule
-          @attachAutocompleteToNewEditorViews()
+
+    # can't use the atom.services.provide API to get autocomplete,
+    # because we call registerProviderForEditor for each editor window,
+    # rather than using a single provider for everything.  so, we need access to the autocomplete module.
+    acPackage = atom.packages.getActivePackage("autocomplete-plus")
```

Comment:
Do not activate autocomplete-plus. The current editor is passed in the options object to your requestHandler (`options.editor`). Use the services approach instead.